### DISCUSSION TITLE
debrid

### DISCUSSION THREAD

**Discussion by jdnewland** [itsToggle/plex_debrid#80] [Q&A] [answered]
plex_debrid does not see my library. It was working and I changed some settings. Trying it again and no matter what I do I can't get it to see my plex library. I have things in my library. Is there a way to reset the script completely?

I'm using your version of rclone. newest one.

real debrid premium is set up.

plex library stuff here. 

![debridplexerror](https://user-images.githubusercontent.com/5415055/185780390-1c723512-38cc-471a-b9ba-ec61e5743ec0.png)
![rclone](https://user-images.githubusercontent.com/5415055/185780407-b0282220-b58b-498a-8f17-a7eb4a5e831b.png)
![plexremote](https://user-images.githubusercontent.com/5415055/185780455-d4afdc13-d0a1-4152-99b9-c609202d3fe6.png)
![library address](https://user-images.githubusercontent.com/5415055/185780524-e84efb64-857b-4d46-805e-8b942e4727da.png)

**Comment by itsToggle** [comment]
I should probably implement a check for correct url format..

You need to add the protocoll "http://" before your IP address:)

There is also a "space" character at the end of the url, which you should remove.

You can always reset your settings by renaming the "settings.json" file to something else. plex_debrid will guide you through the initial setup again.

**Comment by jdnewland** [comment, marked as answer]
Thank you! I actually fixed it right before you replied by adding the ip address to the config.jason file with the http. But you're right it was definitely that!

**Comment by jdnewland** [comment]
And thanks for the script, it's really cool.
